features/frameworks/greentea-client/source/greentea_metrics.cpp

```diff
@@ -65,8 +69,21 @@ void greentea_metrics_report()
     send_stack_info();
     Thread::attach_terminate_hook(NULL);
 #endif
+#if defined(MBED_CPU_STATS_ENABLED)
+    send_CPU_info();
+#endif
 }
 
+#if defined(MBED_CPU_STATS_ENABLED)
+static void send_CPU_info()
+{
+    greentea_send_kv("__cpu_info        up time", mbed_uptime());
```

Comment:
`mbed_uptime` , `mbed_time_sleep` , `mbed_time_deepsleep` are not API's. They are internal functions in Mbed OS. 
Please use `mbed_stats_cpu_get` to get all CPU related info

Reply:
@deepikabhavnani We might want to add that detail to the documentation? These functions are declared in power management header file - thus I would also use it . I do not see any internal detail mentioned there. 

If they are internal, why are there declared then in the power m. header file (we got there _internal functions - intention is clear).

Reply:
@0xc0170 - I was not completely clear with my comment, those functions can be used if required. But since this PR is about adding CPU stats for testing it will be good to use actual API  `mbed_stats_cpu_get` and not the functions called by API.

Reply:
Understood . Still confusing to a user - which one to use? Is it fine as it is, all those functions exposed ?

@jeromecoutant Can you update to use `mbed_stats_cpu_get` rather?

Reply:
User can use mbed_uptime just to know since when the system is up and running (ok to have public).  mbed_stats_cpu_get will give all the cpu related information. We have plans to expand this API to have additional information and parts of which may not be in power management header file.